apps/api/src/app/models/courses/courses.service.ts

```diff
@@ -22,7 +22,7 @@ export class CoursesModelService {
     CoursesViewModel
   >;
 
-  perPage: number = 25;
+  perPage: number = 5;
   page: number = 1;
```

Comment:
Default variables should be contain in env

Reply:
This shouldn't have been committed in the first place. I told to change this parameter just for dev review.